adaptived/src/effects/cgroup_setting.c

```diff
@@ -74,6 +74,10 @@ static int _cgroup_setting_init(struct adaptived_effect * const eff, struct json
 	if (ret)
 		goto error;
 
+	ret = adaptived_file_exists(setting_str);
```

Comment:
Also if we're going to check if the file exists, I think we need a way for the user to skip this check.

What if they had a config like this:
```
    causes:
        always
    effects:
        make_a_cgroup
        operate_on_the_cgroup_just_made
```

A config like the above wouldn't work with the checks as written because the cgroup doesn't exist until the `make_a_cgroup`'s main runs.

Reply:
We could mimic the `-ENOENT` logic for other optional flags